### PR TITLE
Adding documentationon how to run openssh containers as root

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25853
fixes: https://github.com/demisto/etc/issues/25851
## Description
Since this integration runs ssh commands which require a user that is valid in both container and host scope- Adding documentation oh how to run openssh images as root user when non-root internal user is configured.

### CODE REVIEW

**Comment by Itay4** [comment]
lets add it to the readme, not the detailed description

**Comment by hod-alpert** [comment]
@Itay4 Moved

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/README.md [hunk 1]
```diff
@@ -76,6 +76,13 @@
     Click&nbsp;<strong>Test</strong>&nbsp;to validate the new instance.
   </li>
 </ol>
+<ul>
```
Comment (Itay4) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/README.md:79]: lets move it to the top of the doc, above the playbook section

**Comment by hod-alpert** [comment]
Like this?

**Comment by Itay4** [comment]
@kirbles19 please review

**Comment by kirbles19** [comment]
@hod-alpert Done, no changes required.

**Pull request merged**